modules/Bio/EnsEMBL/DBSQL/GeneAdaptor.pm

```diff
@@ -1200,7 +1200,7 @@ sub store_alt_alleles {
       if ($gene->slice->is_reference()) {
           $type{'IS_REPRESENTATIVE'} = 1;
       }
-      push @$aa_record, \%type; 
+      push @$aa_record, \%type;
```

Comment:
At this point I am now bored of scanning for substantive changes amongst all the deletions of trailing whitespace.

Reply:
I know, promise from now on I'll do the whitespace removal in separate commits.
But admit it, looks sooo much better (even if you can't see it because it's whitespace!).